tests/data/dataconverter/readers/mpes/config_file.json

```diff
@@ -14,328 +14,399 @@
-  "/ENTRY[entry]/PROCESS[process]/REGISTRATION[registration]/TRANSFORMATIONS[tranformations]/AXISNAME[rot_z]/@offset": "@attrs:metadata/momentum_correction/adjust_params/offset",
-  "/ENTRY[entry]/PROCESS[process]/REGISTRATION[registration]/TRANSFORMATIONS[tranformations]/AXISNAME[rot_z]/@vector": "@attrs:metadata/momentum_correction/adjust_params/rotation_vector",
-  "/ENTRY[entry]/PROCESS[process]/REGISTRATION[registration]/TRANSFORMATIONS[tranformations]/AXISNAME[rot_z]/@depends_on": "trans_y",
-  "/ENTRY[entry]/PROCESS[process]/energy_calibration/applied": true,
-  "/ENTRY[entry]/PROCESS[process]/energy_calibration/coefficients": "@attrs:metadata/energy_correction/calibration/coefficients",
-  "/ENTRY[entry]/PROCESS[process]/energy_calibration/fit_function": "@attrs:metadata/energy_correction/calibration/fit_function",
-  "/ENTRY[entry]/PROCESS[process]/energy_calibration/original_axis": "@attrs:metadata/energy_correction/tof",
-  "/ENTRY[entry]/PROCESS[process]/energy_calibration/calibrated_axis": "@attrs:metadata/energy_correction/calibration/axis",
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[kx_calibration]/applied": true,
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[kx_calibration]/scaling": "@attrs:metadata/momentum_correction/calibration/scale_kx",
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[kx_calibration]/offset": "@attrs:metadata/momentum_correction/offset_kx",
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[kx_calibration]/calibrated_axis": "@attrs:metadata/momentum_correction/calibration/axis_kx",
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[ky_calibration]/applied": true,
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[ky_calibration]/scaling": "@attrs:metadata/momentum_correction/calibration/scale_ky",
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[ky_calibration]/offset": "@attrs:metadata/momentum_correction/offset_ky",
-  "/ENTRY[entry]/PROCESS[process]/CALIBRATION[ky_calibration]/calibrated_axis": "@attrs:metadata/momentum_correction/calibration/axis_ky",
-  "/ENTRY[entry]/DATA[data]/@axes": "@data:dims",
-  "/ENTRY[entry]/DATA[data]/AXISNAME_indices[@*_indices]": "@data:*.index",
-  "/ENTRY[entry]/DATA[data]/@signal": "data",
-  "/ENTRY[entry]/DATA[data]/data": "@data:data",
-  "/ENTRY[entry]/DATA[data]/data/@units": "counts",
-  "/ENTRY[entry]/DATA[data]/AXISNAME[*]": "@data:*.data",
-  "/ENTRY[entry]/DATA[data]/AXISNAME[*]/@units": "@data:*.unit"
+  "/ENTRY[entry]/INSTRUMENT[instrument]/ELECTRONANALYSER[electronanalyser]/COLLECTIONCOLUMN[collectioncolumn]": {
+    "projection": "@attrs:metadata/instrument/analyzer/projection",
+    "scheme": "angular dispersive",
+    "mode": "@attrs:metadata/instrument/analyzer/lens_mode",
+    "extractor_voltage": "@attrs:metadata/file/KTOF:Lens:Extr:V",
+    "extractor_voltage/@units": "V",
+    "extractor_current": "@attrs:metadata/file/KTOF:Lens:Extr:I",
+    "extractor_current/@units": "µA",

```

Comment:
I would say that it would be best to be consistent, yes. But I think this involves a bit more of a discussion if we generally want this. I'll create an issue for this and we keep it in the agenda for discussing after the workshop (+ we will also discuss about this at the workshop).

Reply:
I created https://github.com/FAIRmat-NFDI/nexus_definitions/issues/148 for this.

Reply:
Yes, let's discuss this at the workshop. There was also the request to e.g. have a calibration documented with each of these values, I think we did not address this yet, no?